lib/ansible/executor/task_executor.py

```diff
@@ -758,9 +758,14 @@ def _get_action_handler(self, connection, templar):
         Returns the correct action plugin to handle the requestion task action
         '''
 
+        network_group_modules = frozenset(['eos', 'nxos', 'ios', 'iosxr', 'junos', 'vyos'])
```

Comment:
Given where we are in the dev cycle, this gets us through 2.3 and then we need to commit to finding a better long term solution.  The drawback to not doing this is that every single network module will need a local action in 2.3